wellregistry/registry/tests/admin/test_monitoring_location.py

```diff
@@ -150,6 +153,13 @@ def test_changelist_view_with_usgs_user(self):
         self.assertNotIn(b'Add monitoring location', resp.content)
         self.assertNotIn(b'Bulk Upload', resp.content)
 
+    def test_site_no_auto_complete_View(self):
```

Comment:
This should be moved to 'wellregistry/registry/tests/admin/test_auto_complete.py. The tests for a unit should follow the same directory structure as the application code. 

In addition this test should be all lowercase snakecase.

Reply:
fixed

Reply:
fixed